beacon-chain/core/time/slot_epoch.go

```diff
@@ -48,7 +48,7 @@ func NextEpoch(state state.ReadOnlyBeaconState) types.Epoch {
 
 // AltairCompatible returns if the input state `s` is altair compatible and input epoch `e` is higher equal than fork epoch.
 func AltairCompatible(s state.BeaconState, e types.Epoch) bool {
-	return s.Version() == version.Altair && e >= params.BeaconConfig().AltairForkEpoch
+	return s.Version() >= version.Altair && e >= params.BeaconConfig().AltairForkEpoch
```

Comment:
> AltairCompatible returns if the input state `s` is altair compatible

Can we say with 100% confidence that a bellatrix state is altair compatible? I just want to make sure this is safe. If yes, then please add a test case for bellatrix state + altair fork epoch

Reply:
Yes. We currently can assume that future forks encompass features of previous forks. Until such an assumption goes away, this is valid. The good thing is even if the premise is broken, it won't fail silently

Reply:
Share radek's concerns here, this function mentions being `AltairCompatible` , which would indicate to the reader it is only to be used in altair. If we want to denote by function then renaming this to `SyncCommitteeCompatible ` or some other name that is more indicative what this function is really checking for.

Reply:
I'd think the naming implies. Compatible doesn't mean it's "only" used rather than it is the subset of the input. In this case, Bellatrix and the future hardfork being the super set where altair is the subset. 

I'm happy to change it to something else though since you asked!

Reply:
There could be a future HF, where sync committees are removed then the phrase 'altair compatible' wouldn't indicate a subset then.

> I'm happy to change it to something else though since you asked!

Yeap, I think changing the naming would address my main concern over here

Reply:
Renamed to `HigherThanAltairVersionAndEpoch`. Hopefully that is better!

Reply:
One more nit, shouldn't it be :
```
HigherThanOrEqualTo
```
since we check for `>=`